apps/server/src/shared/domain/entity/boardnode/boardnode.entity.ts

```diff
@@ -8,6 +8,7 @@ import { BoardDoBuilder, BoardNodeType } from './types';
 const PATH_SEPARATOR = ',';
 
 @Entity({ tableName: 'boardnodes', discriminatorColumn: 'type', abstract: true })
+@Index({ properties: ['path', 'type'] })
```

Comment:
so this would be a compound index, right? to my understanding, this is useful when we user queries for both, but if we query only for type or only for path, should be separated

Reply:
yes, you are right. Fixed